NorthstarDLL/engine/r2engine.h

```diff
@@ -173,20 +173,29 @@ namespace R2
 		CHANGELEVEL = 9, // server is changing level; please wait
 	};
 
-	// clang-format off
-	OFFSET_STRUCT(CBaseClient)
+#pragma pack(push, 1)
+	typedef struct
 	{
-		STRUCT_SIZE(0x2D728)
-		FIELD(0x16, char m_Name[64])
-		FIELD(0x258, KeyValues* m_ConVars)
-		FIELD(0x2A0, eSignonState m_Signon)
-		FIELD(0x358, char m_ClanTag[16])
-		FIELD(0x484, bool m_bFakePlayer)
-		FIELD(0x4A0, ePersistenceReady m_iPersistenceReady)
-		FIELD(0x4FA, char m_PersistenceBuffer[PERSISTENCE_MAX_SIZE])
-		FIELD(0xF500, char m_UID[32])
-	};
-	// clang-format on
+		char _unk1[22]; // 0x0 ( Size: 22 )
```

Comment:
correct

Reply:
oh, `unk` is `unknown`, lol

Reply:
Actually is `unk` a common abbreviation for `unknown` that I'm unaware of? Or is there a reason to use `unk` instead of the more self-explanatory `unknown`? ^^"

Reply:
idk.
I picked it up while digging through decompiles and it stuck with me.
Used it here because it was nice and small

Reply:
> Actually is `unk` a common abbreviation for `unknown` that I'm unaware of? Or is there a reason to use `unk` instead of the more self-explanatory `unknown`? ^^"

I've seen both, I think it changes based on the decompiler used. `unk` is used over `unknown` because it's more concise, there isn't really much meaning behind saying that something is "unknown", the important part of the name is the offset that is appended